tests/unit/iontests.js

```diff
@@ -150,6 +150,10 @@ define([
             assert(r1.annotations() !== undefined, 'annotations() is udnefined');
```

Comment:
Not related, but 'udnefined' caught my eye.

Reply:
**udnefined**, *adjective*:  not defined and scrambled.  Usage:  *"I prefer my eggs udnefined"*

fixed -- thank you